### ISSUE TITLE
Doubt

### ISSUE THREAD

**Issue by mirraaj** [chainer/chainerrl#283]
Could you please let me know why there is a negative sign? 

https://github.com/chainer/chainerrl/blob/e424a75c3c7560fb6d78594e70f52af9668788fb/chainerrl/agents/acer.py#L225

**Comment by muupan** [comment]
That's because 
```
loss.backward()
optimizer.update()
```
will try to minimize `loss` (θ -= lr * ∇_θ loss) in Chainer.

**Comment by mirraaj** [comment]
@muupan. During the policy gradient approach, we try to do θ += lr * ∇_(θ)(π(θ)) * A. In acer the gradient is being subtracted by derivative of trust region optimization (equation 12 (https://arxiv.org/pdf/1611.01224.pdf)). Since we have defined loss function to be a function of π(θ), we do loss = - single_step_loss, to allow SGD to do its work. However, if you look closely, for SGD we need to add the second term instead of subtracting(to make it compatible for SGD). Since you people have already taken care of that in the line 241, you do not need to put the (-kl) term.  https://github.com/chainer/chainerrl/blob/e424a75c3c7560fb6d78594e70f52af9668788fb/chainerrl/agents/acer.py#L241

The (+kl) term should be used for computing since we need the gradient of the parameters and that is independent of the optimizer. Let me know how you feel about this.
